README.md

```diff
@@ -154,31 +129,26 @@ There are a few known limitations with the bundled TypeScript definitions at thi
  * Service client typings reflect the latest `apiVersion`, regardless of which `apiVersion` is specified when creating a client.
  * Service-bound parameters use the `any` type.
 
-## Getting Help
-Please use these community resources for getting help. We use the GitHub issues for tracking bugs and feature requests and have limited bandwidth to address them.
+# Getting Help
 
- * Ask a question on [StackOverflow](https://stackoverflow.com/) and tag it with `aws-sdk-js`
- * Come join the AWS JavaScript community on [gitter](https://gitter.im/aws/aws-sdk-js?source=orgpage)
- * Open a support ticket with [AWS Support](https://console.aws.amazon.com/support/home#/)
- * If it turns out that you may have found a bug, please [open an issue](https://github.com/aws/aws-sdk-js/issues/new)
+The best way to interact with our team is through GitHub. You can [open an issue](https://github.com/aws/aws-sdk-js/issues/new/choose) and choose from one of our templates for [bug reports](https://github.com/aws/aws-sdk-js/issues/new?assignees=&labels=bug%2C+needs-triage&template=---bug-report.md&title=), [feature requests](https://github.com/aws/aws-sdk-js/issues/new?assignees=&labels=feature-request&template=---feature-request.md&title=) or [guidance](https://github.com/aws/aws-sdk-js/issues/new?assignees=&labels=guidance%2C+needs-triage&template=---questions---help.md&title=). You may also find help on community resources such as [StackOverFlow](https://stackoverflow.com/questions/tagged/aws-sdk-js) with the tag #aws-sdk-js or If you have a support plan with [AWS Support](https://aws.amazon.com/premiumsupport/), you can also create a new support case.
```

Comment:
Spread the content over multiple lines so that it's easy to review. Markdown will display content as continuous lines. Ditto for other sections where long paragraphs are used.

```suggestion
The best way to interact with our team is through GitHub.
You can [open an issue](https://github.com/aws/aws-sdk-js/issues/new/choose) and choose from one of our templates for
[bug reports](https://github.com/aws/aws-sdk-js/issues/new?assignees=&labels=bug%2C+needs-triage&template=---bug-report.md&title=),
[feature requests](https://github.com/aws/aws-sdk-js/issues/new?assignees=&labels=feature-request&template=---feature-request.md&title=)
or [guidance](https://github.com/aws/aws-sdk-js/issues/new?assignees=&labels=guidance%2C+needs-triage&template=---questions---help.md&title=).
You may also find help on community resources such as [StackOverFlow](https://stackoverflow.com/questions/tagged/aws-sdk-js) with the tag #aws-sdk-js
or If you have a support plan with [AWS Support](https://aws.amazon.com/premiumsupport/), you can also create a new support case.
```

Reply:
+1

Reply:
Thanks for that, i changed that